ui/appui-abstract/src/appui-abstract/backstage/BackstageItemsManager.ts

```diff
@@ -15,6 +15,8 @@ import { BackstageItem } from "./BackstageItem";
 
 /** Arguments of [[BackstageItemsManager.onChanged]] event.
  * @internal
+ * @deprecated in 4.10.x. Use `BackstageItemsChangedArgs` from `@itwin/appui-react`.
```

Comment:
Alternatively, add an event to `FrameworkBackstage`

```suggestion
 * @deprecated in 4.10.x. Used internally by `@itwin/appui-react`.
```